stores/useComicStore.js

```diff
@@ -8,8 +8,17 @@ import mitt from 'mitt';
 export const useComicStore = defineStore('comic', () => {
     const comic = new Comic(null, null, null);
     let currentElement = ref(null);
+    let elementMap = ref(null);
     let bus = mitt();
 
+    function setElementMap(map) {
+        elementMap.value = map;
+    }
+
+    function getElementMap() {
+        return elementMap;
+    }
+
```

Comment:
Rather than saving the active elements globally, it is a better approach to pass them to the component (`LayerObject`) over a prop, since this removes dependencies.